packages/scripts/utils/config.js

```diff
@@ -24,6 +24,16 @@ const hasBabelConfig = () =>
 	hasProjectFile( '.babelrc' ) ||
 	hasPackageProp( 'babel' );
 
+// See https://cssnano.co/docs/config-file.
+const hasCssnanoConfig = () =>
```

Comment:
I discovered in the source code that the list of supported file names is longer that in the docs.